<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMonitoring</class>
 <widget class="QWidget" name="WidgetMonitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="tvInputs1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="tvInputs2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>221</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="tvInputs3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>221</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>671</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lDateTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00-00-00 00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lUptime">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Uptime 00:00:00</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSetTime">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/time.png</normaloff>:/icons/time.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
